.github/workflows/release-suite-message-system-config.yml

```
@@ -29,7 +29,7 @@ jobs:
         uses: aws-actions/configure-aws-credentials@v4
         with:
           role-to-assume: ${{ env.ROLE_TO_ASSUME  }}
-          aws-region: ${{ env.AWS_REGION }}
+          aws-region: ${{ env.AWS_R EGION }}
```

actionlint (1.7.4)

parser did not reach end of input after parsing the expression. 1 remaining token(s) in the input: "IDENT"

(expression)